File: CS/CheckAsYouTypeInXtraGrid/Record.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace CheckAsYouTypeInXtraGrid
{
    public class Record
    {
        private int _Id;
        private string _Note;
        private DateTime _TimeMark;

        public Record(int id, string note, DateTime timemark)
        {
            this._Id = id;
            this._Note = note;
            this._TimeMark = timemark;
        }                                   

        public DateTime TimeMark
        {
            get { return _TimeMark; }
            set { _TimeMark = value; }
        }

        public int Id
        {
            get { return _Id; }
            set { _Id = value; }
        }

        public string Note
        {
            get { return _Note; }
            set { _Note = value; }
        }
    }
}

File: VB/CheckAsYouTypeInXtraGrid/Record.vb
Imports System
Imports System.Collections.Generic
Imports System.Text

Namespace CheckAsYouTypeInXtraGrid
    Public Class Record
        Private _Id As Integer
        Private _Note As String
        Private _TimeMark As Date

        Public Sub New(ByVal id As Integer, ByVal note As String, ByVal timemark As Date)
            Me._Id = id
            Me._Note = note
            Me._TimeMark = timemark
        End Sub

        Public Property TimeMark() As Date
            Get
                Return _TimeMark
            End Get
            Set(ByVal value As Date)
                _TimeMark = value
            End Set
        End Property

        Public Property Id() As Integer
            Get
                Return _Id
            End Get
            Set(ByVal value As Integer)
                _Id = value
            End Set
        End Property

        Public Property Note() As String
            Get
                Return _Note
            End Get
            Set(ByVal value As String)
                _Note = value
            End Set
        End Property
    End Class
End Namespace
